src/store/reducers/stateCache/stateCache.ts

```diff
@@ -21,7 +21,7 @@ const initialState: StateCacheProps = {
     passwordIsSkipped: true,
   },
   currentOperation: OperationType.IDLE,
-  queueConnectionCredentialRequest: {
+  queueIncomingRequest: {
     isProcessing: false,
```

Comment:
can `isProcessing` be true and `isPaused` also true at the same time?

Reply:
@jimcase I didn't create that, all I have done in this PR was renaming it to match the new name, but you can investigate that if you want.

Reply:
`isPaused` is used when the device is locked, so that connection requests don't come in and overlay the passcode login.

`isProcessing` is used to track if there are still more incoming requests in the queue as there may be multiple, so this is used once the app is unlocked. So the queue gets popped every 0.5 seconds or something.

As far as I remember, `isProcessing` gets auto set to `false` if we are paused, but I do remember at the time at least both variables were needed. Not sure if it can be refactored now, but Sal shouldn't need to do that for this PR. @Sotatek-HocNguyen please feel free to chime in :)